app/helpers/application_helper.rb

```diff
@@ -72,4 +72,29 @@ def colored_span(color, content)
       content
     end
   end
+
+ def easter
+    year = Date.today.year
+    a = year % 19
+    b = (year/100).floor
+    c = year % 100
+    d = (b/4).floor
+    e = b % 4
+    f = ((b+8)/25).floor
+    g = (b-f+1)/3.floor
```

Comment:
Surrounding space missing for operator '/'.<br>Surrounding space missing for operator '+'.<br>Surrounding space missing for operator '-'.